yarn-project/boxes/blank-react/src/tests/blank.contract.test.ts

```diff
@@ -0,0 +1,72 @@
+import { BlankContract } from '../artifacts/blank.js';
+import { rpcClient } from '../config.js';
+import { callContractFunction, deployContract, getWallet } from '../scripts/index.js';
+import {
+  AccountWallet,
+  AztecAddress,
+  AztecRPC,
+  CompleteAddress,
+  Contract,
+  Fr,
+  Wallet,
+  createAztecRpcClient,
+  waitForSandbox,
+} from '@aztec/aztec.js';
+import { createDebugLogger } from '@aztec/foundation/log';
+
+const logger = createDebugLogger('aztec:http-rpc-client');
+
+// assumes sandbox is running locally, which this script does not trigger
+// as well as anvil.  anvil can be started with yarn test:integration
+const setupSandbox = async () => {
+  const { SANDBOX_URL = 'http://localhost:8080' } = process.env;
+  const aztecRpc = createAztecRpcClient(SANDBOX_URL);
+  await waitForSandbox(aztecRpc);
+  return aztecRpc;
+};
+
+async function deployZKContract(owner: CompleteAddress, wallet: Wallet, rpcClient: AztecRPC) {
+  logger('Deploying Blank contract...');
+  const contractAddress = await deployContract(owner, BlankContract.abi, [], Fr.random(), rpcClient);
+
+  logger(`L2 contract deployed at ${contractAddress}`);
+  const contract = await BlankContract.at(contractAddress, wallet);
+  return contract;
+}
+
+async function call(contractAddress: AztecAddress, testTokenContract: Contract, address: CompleteAddress) {
```

Comment:
Probably don't need a separate function for doing this one thing. I have to scroll up and down to see what the test is doing. Will be easier to read if it's in the test.